form/button.ftd

```diff
@@ -18,8 +15,8 @@
 -- or-type button-variant:
 
 -- constant integer solid: 0
--- constant integer outline: 1
--- constant integer ghost: 2
+-- constant integer pressed: 1
```

Comment:
`pressed` is not a proper variant. This is a UI state, an internal detail. No one creates a button in pressed state, "oh on login page we will show a button that is already pressed!". On login page you ahve a solid or outline or ghost button, and *each* of them have pressed state. Do you think a solid button will never be pressed?